### PR TITLE
Update tags

### PR DESCRIPTION
**Pull request by rmenner**
# Alaska Airlines Pull Request

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (please elaborate)

linking missing version tags

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Update documentation, dependencies, tests, and configuration files. Update the contributing guidelines to clarify the process for submitting issues and pull requests.  Update the code of conduct to remove specific contact information.  Update the changelog to reflect the latest changes.  Update the package version to 3.1.0.

Documentation:
- Update documentation paths and content, including API examples and the README.

Tests:
- Update test workflows to use Node.js versions 20 and 22.

Chores:
- Update repository settings, labels, and stale bot configuration.

### CODE REVIEW

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
Seems like the work that was done in this release has been undone: https://github.com/AlaskaAirlines/auro-sidenav/compare/v3.0.1...v3.1.0

Is there a reason for this or was this an accident?

**Comment by rmenner** [comment]
> Seems like the work that was done in this release has been undone: [v3.0.1...v3.1.0](https://github.com/AlaskaAirlines/auro-sidenav/compare/v3.0.1...v3.1.0)
> 
> Is there a reason for this or was this an accident?

Nice catch. I will make sure those updates are transferred over.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,80 +1,5 @@
 # Semantic Release Automated Changelog
 
-# [3.1.0](https://github.com/AlaskaAirlines/auro-sidenav/compare/v3.0.1...v3.1.0) (2024-12-22)
```
Comment (jordanjones243) [inline CHANGELOG.md:3]: Need to re-instate the changelog comments

**Review by jordanjones243 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rmenner) [reply CHANGELOG.md:3]: All the comments are there. If it looks like they are removed. It's because they moved around a little.

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**